plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx

```diff
@@ -54,29 +54,38 @@ const useStyles = makeStyles((theme: Theme) =>
   }),
 );
 
-const copyToClipboard = ({ about }: { about: DevToolsInfo | undefined }) => {
-  if (about) {
-    let formatted = `OS: ${about.operatingSystem}\nResources: ${about.resourceUtilization}\nnode: ${about.nodeJsVersion}\nbackstage: ${about.backstageVersion}\nDependencies:\n`;
-    const deps = about.dependencies;
-    for (const key in deps) {
-      if (Object.prototype.hasOwnProperty.call(deps, key)) {
-        formatted = `${formatted}    ${deps[key].name}: ${deps[key].versions}\n`;
-      }
-    }
-    window.navigator.clipboard.writeText(formatted);
-  }
-};
-
 /** @public */
 export const InfoContent = () => {
   const classes = useStyles();
+  const alertApi = useApi(alertApiRef);
+  const [infoCopied, setInfoCopied] = useState(false);
   const { about, loading, error } = useInfo();
 
+  const copyToClipboard = () => {
+    if (about) {
+      let formatted = `OS: ${about.operatingSystem}\nResources: ${about.resourceUtilization}\nnode: ${about.nodeJsVersion}\nbackstage: ${about.backstageVersion}\nDependencies:\n`;
+      const deps = about.dependencies;
+      for (const key in deps) {
+        if (Object.prototype.hasOwnProperty.call(deps, key)) {
+          formatted = `${formatted}    ${deps[key].name}: ${deps[key].versions}\n`;
+        }
+      }
+      window.navigator.clipboard.writeText(formatted);
```

Comment:
Please use this instead: `import useCopyToClipboard from 'react-use/lib/useCopyToClipboard'`. It's kinda hard to get right and requires a fair bit of code, so we prefer that utility instead.

Reply:
Oh, cool, didn't know about that! This is mostly my original code moved around so any further improvements would be great 👍